samples/SocketsSample/Startup.cs

```diff
@@ -18,11 +19,12 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddSockets();
 
-            services.AddSignalR(options =>
-            {
-                // Faster pings for testing
-                options.KeepAliveInterval = TimeSpan.FromSeconds(5);
-            });
+            services
```

Comment:
I liked the previous formatting

Reply:
The `AddRedis` line makes it look weird, particularly if you uncomment it. I don't really mind either way though.